proactor/src/main/java/com/iluwatar/proactor/Client.java

```diff
@@ -0,0 +1,52 @@
+package com.iluwatar.proactor;
+
+import java.io.NotActiveException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Client side.
+ */
+public class Client {
+
+  /**
+   *  Logger.
+   */
+  private static final Logger LOGGER = LoggerFactory.getLogger(Client.class);
+
+  /**
+   * Normally use internet to communicate between server and client.
+   */
```

Comment:
The comment is not very useful. Either rephrase or remove.

Reply:
remove

Reply:
Alright, to pass Sonar CI, have to write something